wscript

```diff
@@ -237,3 +239,15 @@ def build_server(bld):
                     'MODULE_VERSION="%s"' % VERSION
                 ],
                 use=['stechec2', 'gflags'])
+
+def build_replay(bld):
+    bld.program(source='''
+            src/replay/main.cc
+            src/replay/replay.cc
+        ''',
+                target='stechec2-replay',
+                defines=[
+                    'MODULE_COLOR=ANSI_COL_RED', 'MODULE_NAME="replay"',
```

Comment:
Another color maybe? This is already used by the server.

Reply:
The idea behind stechec2-replay is that it is exactly like stechec2-server, just without clients.

It's just implemented as a separate bin here for simplicity, but another implem could have been to re-use the same stechec2-server and feed the replayed actions with a new "stechec2-replay-client" instead.